images/win/scripts/Installers/Validate-SQLExpress.ps1

```diff
@@ -0,0 +1,15 @@
+$sqlConn = New-Object System.Data.SqlClient.SqlConnection
```

Comment:
Please update documentation where you should provide:
1. SQL Version
2. InstanceName
3. UserName
4. Password
5. Default path

Reply:
Added Markdown. Please review